rosdep/python.yaml

```
@@ -8726,6 +8726,13 @@ python3-pyxdg-pip:
   ubuntu:
     pip:
       packages: [pyxdg]
+python3-qrcode:
+  arch: [python-qrcode]
+  debian: [python3-qrcode]
+  fedora: [python3-qrcode]
+  gentoo: [dev-python/qrcode]
+  nixos: [python39Packages.qrcode]
+  ubuntu: [python3-qrcode]
```

Consider adding additional definitions for opensuse and rhel as suggested by the linter.

```suggestion
python3-qrcode:
  arch: [python-qrcode]
  debian: [python3-qrcode]
  fedora: [python3-qrcode]
  gentoo: [dev-python/qrcode]
  nixos: [python39Packages.qrcode]
  opensuse: [python3-qrcode]
  rhel: [python3-qrcode]
  ubuntu: [python3-qrcode]
```